src/codegen/llvm/Builder.zig

```diff
@@ -14296,6 +14292,51 @@ pub fn toBitcode(self: *Builder, allocator: Allocator) bitcode_writer.Error![]co
                 });
             }
 
+            // Write global attached metadata
+            {
+                for (self.variables.items) |variable| {
```

Comment:
Is there a reason to not just iterate `self.globals` here?

Reply:
Not really, I could change that. The only reason I do them separately is that function metadata should for declarations this way (for definitions it's emitted inside the function block). But I could just as well check for functions specifically.